samples/LoggingTracer/LoggingTracer/LoggingTextFormat.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using OpenTelemetry.Trace;
+using OpenTelemetry.Context.Propagation;
+
+namespace LoggingTracer
+{
+    public sealed class LoggingTextFormat : ITextFormat
+    {
+        public ISet<string> Fields => throw new NotImplementedException();
```

Comment:
if the purpose is to demonstrate all callback that may be called - this should log and return an empty collection